azure-pipelines.yml

```diff
@@ -6,21 +6,68 @@
 trigger:
 - master
 
+variables:
+  cli11.single: ON
+  cli11.std: 14
+  cli11.build_type: Debug
+  cli11.options:
+
 jobs:
-- job: Linux
+
+- job: ClangFormat
   pool:
     vmImage: 'ubuntu-16.04'
   steps:
-    - template: .ci/azure-steps.yml
+    - script: scripts/check_style_docker.sh
```

Comment:
Will the other format check be dropped.  Seems a little redundant to have both, or are they checking different things?

Reply:
I'll be slowly reducing our usage of non-Azure CI, so yes. I could probably go ahead and drop that in this PR.

Reply:
I'm actually still only checking "clang-tidy" on Travis. I'll need to move that in here in the future.